tests/frontend_selenium/tests/test_twin.py

```diff
@@ -22,119 +22,68 @@ def before_test_setup(browser, create_account):
 def test_accept_terms_conditions(browser):
     """
       Test Case: TC924 - Accept terms and conditions
-      Test Cases: TC931- button why do I even need twin
       Steps:
           - Navigate to dashboard.
           - Click on the desired account from the dashboard homepage.
           - Click on Accept The Terms and Conditions.
           - Use polka password authentication.
-          - Click on button why do I even need a twin.
       Result: Open same account on dashboard homepage and assert that no terms to accept when you come back to this account twin page.
-              Assert that it will go to righ link.
     """
     twin_page, polka_page, password = before_test_setup(browser, True)
     twin_page.accept_terms_conditions()
     polka_page.authenticate_with_pass(password)
-    assert twin_page.wait_for('Planetary using Yggdrasil IPV6')
-    assert 'Planetary using Yggdrasil IPV6' in browser.page_source
-    assert twin_page.Button_why_doIeven_need_twin(
-    ) == 'https://library.threefold.me/info/manual/#/manual__yggdrasil_client'
+    assert twin_page.wait_for('Accepted!')
+    assert twin_page.wait_for('Choose a Relay Address')
 
 
-def test_create_twin_IP(browser):
+def test_create_twin_relay(browser):
```

Comment:
2 Tests in the test_twin file failed as seen below.

![image](https://user-images.githubusercontent.com/101194226/223752117-b13798f2-953a-436b-a792-a37064712157.png)

full logs can be found [here](https://gist.github.com/mohamedamer453/d319fbccf48605359283d57b0e56e6f8).

Reply:
The issue was resolved and the `edit_twin_relay` test doesn't fail anymore